docs/packages.rst

```diff
@@ -46,4 +45,3 @@ fv3config_ (`docs <https://fv3config.readthedocs.io/en/latest/>`_) provides rout
     readme_links/synth_readme
```

Comment:
Nit: OK this is really not related to this PR but since we're in the docs can you tweak the README.mds in `loaders` and `fv3viz` so they both have a single proper markdown header? eg., 
```
fv3fviz
======
```
and 
```
loaders
=======
```
If you look at the [table of contents on the packages page](http://vulcanclimatemodeling.com/docs/fv3net/packages.html) of our docs, right now loaders has two (one for batches and one for mappers) and fv3viz has none. It's bugged me forever but I've never remembered to get it merged.